### ISSUE TITLE
Xiaomi keyboard ctrl+enter (switch language) not working on Android

### ISSUE THREAD

**Issue by Theog75** [johannesjo/super-productivity#2784]
### Android

When working with super productivity on Xiaomi pad 6 I can switch language on all apps except super productivity.

- Version used: latest
- Operating System and version: Android 13

### Expected Behavior

language should change when using external keyboard (CTRL+SPACE)

### Current Behavior

cannot change language (only in super productivity all other apps are switching language on ctrl+space shortcut)
### Steps to Reproduce (for bugs)

this error happens every time

### Can you reproduce this reliably?

yes

### Error Log (Desktop only)
no log is available

**Comment by Theog75** [comment]
can anyone help pleasE?

**Comment by CyberLynxAdmin** [comment]
The only sort of "easy" workaround I found while working with apps that expose the issue:

1) press cmd+N to open up notification shade

2) press ctrl+space to change the language (you may notice little popup behind the blur as language changed)

3) touch the screen to get back to the app to contunue editing with language switched to

source: https://www.reddit.com/r/Xiaomi/comments/12bec48/xiaomi_pad_5_dont_change_keyboard_language_with/

**Comment by CyberLynxAdmin** [comment]
Found an actual solution. Now I can change lang at every app!!

Download from Google Store the "**External Keyboard Helper Pro**". It costs **1.49$**.

- Select it as your current keyboard from Languages & Input menu.

- Reboot your device.

- Go to Languages & Input again.

- Open "Manage keyboards" and choose "External Keyboard Helper Pro" to open
setup.

- Go to "Advanced settings" / "Languages switching".

- Click on the "Language-switch key

- Uncheck the "Disabled"

- Choose Swift or Alt checkbox.

- and finally use "Scancode" to choose your preferable key combination and click
"SAVE" / I choose spacebar.

**Comment by vanarok** [comment]
related https://forum.obsidian.md/t/bt-keyboard-language-layout-bug-android/66750/2

**Comment by Muhammedingo** [comment]
> This issue has not received any updates in 90 days. Please comment, if this still relevant!

Yes, this is relevant! Language changing inside apps just doesn't seem to work in Xiaomi Pad 6. 
The cmd+N shortcut is better than nothing for now.

**Comment by timuralchin** [comment]
i found quick solution 
1) alt tab
2) ctrl space

**Comment by timuralchin** [comment]
and more simple

func+space

**Comment by Muhammedingo** [comment]
This is solved with a new system update.
